### PR TITLE
Update to Phoenix 0.8

### PR DESCRIPTION
**Pull request by chrismccord**

### CODE REVIEW

**Comment by mroth** [comment]
FYI I am now working towards integrating this in PR #8, if you wish to follow along!

File: README.md [hunk 1]
```diff
@@ -90,12 +90,12 @@ All benchmarks run on a local dev machine are highly suspect.  If you want
...
 ### Benchmarking Phoenix
-Elixir 0.14.2
+Elixir 1.0.3
 
 ```bash
 $ mix do deps.get, compile
 $ MIX_ENV=prod mix compile.protocols
-$ MIX_ENV=prod elixir -pa _build/prod/consolidated -S mix phoenix.start
+$ MIX_ENV=prod elixir -pa _build/prod/consolidated -S mix phoenix.server
 Running Elixir.Benchmarker.Router with Cowboy on port 4000
```
Comment (mroth) [inline README.md]: I don't get this line of output anymore (process starts silently).  Am I doing something wrong?

Comment (chrismccord) [inline README.md]: This is because logging is disabled. So no worries here.

**Comment by mroth** [comment]
Now integrated in master via #8, not sure why GitHub didn't automatically mark this as closed!

**Pull request closed without merging**